modules/gds_ssh_config/files/gds_ssh_config

```diff
@@ -51,3 +51,17 @@ Host jumpbox-2.management.production
 
 Host *.production
   ProxyCommand ssh -e none %r@jumpbox-1.management.production -W $(echo %h | sed 's/\.production$//'):%p
+
+
+# Carrenza Migration ( Staging + Production )
```

Comment:
I think we can remove the word 'Migration', especially since EFG and Licensify will stay in Skyscape.